contracts/cw1-whitelist/src/contract.rs

```diff
@@ -271,7 +270,7 @@ mod tests {
 
         #[test]
         fn freeze() {
-            let original = whitelist::ExecMsg::Freeze {};
+            let original = whitelist::WhitelistExecMsg::Freeze {};
```

Comment:
Why? We don't want to prefix message names - it is doubling information and makes it clunky. To avoid name collisions do it only for the glue message, but in a private namespace so it's hidden from the world. The user should use just `whitelis::ExecMsg`.

Unless the bug in the schemars is so crappy that it depends on original message name?

Reply:
It seems that this bug might be soon fixed though

https://github.com/GREsau/schemars/pull/178